lib/challenge.js

```diff
@@ -30,10 +40,14 @@
   // - a name
   // - an address
 
-var pet_owner;
 
-// Answer Starts Here
 
+// Answer Starts Here
+var pet_owner {
+  clinic:
```

Comment:
Since no value is supplied for this object literal's `clinic` attribute, this is an incomplete key-value pair. The missing comma, which serves to separate key-value pairs, means the interpreter combines the two lines into a syntactically invalid key-value pair: `clinic: owner: [name, address]`. An error will then be thrown when the second colon is read, since a key-value pair is allowed only one colon.